packages/generate-en10168-pdf-template/src/lib/createProductDescription.ts

```diff
@@ -53,7 +53,7 @@ export function createProductDescription(
     ProductDescription,
     'B01' | 'B02' | 'B09' | 'B10' | 'B11' | 'B12' | 'B12' | 'B13' | 'SupplementaryInformation'
   >;
-  const B02ProductNorms = productNorms(productDescription.B02, i18n);
+  const B02ProductNorms = productNorms(productDescription.B02 as ProductNorms, i18n);
```

Comment:
do not force cast, but return early in case B02 is string, which should not happen. But who knows!